lib/directives/toggle.js

```diff
@@ -1,13 +1,13 @@
 const inBrowser = typeof window !== 'undefined';
 
 import target from './_target';
-
 const listen_types = {click: true};
 
 export default {
-    bind(el, binding) {
-        const targets = target(el, binding, listen_types, ({targets, vm}) => {
+    bind(el, binding, vnode) {
+        const targets = target(vnode, binding, listen_types, ({targets, vnode}) => {
             targets.forEach(target => {
+                const vm = vnode.context;
                 vm.$root.$emit('collapse::toggle', target);
```

Comment:
Also in the section below, you could also replace `el.__vue__` with `vnode.context` on line 17, 

and remove the test for `el.__vue__` on line 14